examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -39,7 +41,19 @@ class CHIPCommand : public Command
     using NodeId                 = ::chip::NodeId;
     using PeerAddress            = ::chip::Transport::PeerAddress;
 
-    CHIPCommand(const char * commandName) : Command(commandName) { AddArgument("commissioner-name", &mCommissionerName); }
+    CHIPCommand(const char * commandName, CredentialIssuerCommands * credIssuerCmds = nullptr) : Command(commandName)
+    {
+        AddArgument("commissioner-name", &mCommissionerName);
+
+        if (credIssuerCmds == nullptr)
+        {
+            mCredIssuerCmds = &mExampleCredentialIssuerCmds;
+        }
+        else
+        {
+            mCredIssuerCmds = credIssuerCmds;
+        }
```

Comment:
Unclear why these two cases exist

Reply:
Adding a new constructor with param (CredentialIssuerCommands) will make it clear